### ISSUE TITLE
Curious if this will generate dynamic Javascript proxies

### ISSUE THREAD

**Issue by MikeChristensen** [CoreWCF/CoreWCF#84]
One nice thing about WCF is it has the ability to generate a Javascript proxy script, such as `/MyService.svc/js`.  ASP.NET Core doesn't appear to have anything similar.  Curious if CoreWCF already does (or will) provide this functionality as well.  Thanks!

**Comment by mconnew** [comment]
CoreWCF doesn't do this yet. This isn't on my priority list in the short term, but there's no reason why this can't be done. Can you provide a pointer to how you enable this? If you would be interested in contributing, I can certainly help give pointers and support for porting the code once I have a starting point for enabling the scenario in WCF.

**Comment by mconnew** [comment]
@MikeChristensen, I didn't know much about the dynamic JavaScript proxies feature so I just did a quick deep dive into what's involved. It requires your service to be using WebHttpBinding which is a REST endpoint for your WCF service. I'll be honest, WebHttpBinding will be really far down the list of features being ported by me as CoreWCF is built on top of ASP.NET Core which supports REST natively. Unless you or someone else wants to port WebHttpBinding and WebScriptEnablingBehavior, your better off with an alternative solution. My recommended approach is to port your service to use ASP.NET Core Web API, add an OpenAPI (formerly swagger) endpoint using either [Swashbuckle](https://docs.microsoft.com/en-us/aspnet/core/tutorials/getting-started-with-swashbuckle?view=aspnetcore-5.0) or [NSwag](https://docs.microsoft.com/en-us/aspnet/core/tutorials/getting-started-with-nswag?view=aspnetcore-5.0), then generate a JavaScript client from your OpenAPI file. Here's a project for [code gen linked directly](https://github.com/zijianhuang/openapiclientgen#examples-of-generated-codes) to some examples for generating clients for various JavaScript libraries. There are lots of different JavaScript client generators, this isn't the only one.

**Comment by MikeChristensen** [comment]
Yup, I ended up just kinda hard coding the stuff WCF used to generate for you automatically.  It works fine, just more things to update if your code changes.  This would definitely be a "nice to have", but I understand if there's higher priorities..
